### DISCUSSION TITLE
Expected timeline for the v1.6.x release?

### DISCUSSION THREAD

**Discussion by johanhoglund** [CoreWCF/CoreWCF#1439] [Releases]
Hi,

I just wanted to ask if there is a rough timeline for the v1.6.x release?

Thanks!

**Comment by mconnew** [comment]
I just published them to nuget.org, but they are unlisted until I can finish the release notes, they are going to be quite big this time. If you manually specify 1.6.0 as your version, you can get it.

**Reply by johanhoglund** [reply]
Amazing - thanks!
